ProjectLighthouse/Database.cs

```diff
@@ -53,7 +54,7 @@ public class Database : DbContext
     public DbSet<APIKey> APIKeys { get; set; }
     public DbSet<Playlist> Playlists { get; set; }
     public DbSet<PlatformLinkAttempt> PlatformLinkAttempts { get; set; }
-
+    public DbSet<BlockedProfile> BlockedProfiles { get; set; }
```

Comment:
```suggestion
    public DbSet<BlockedProfile> BlockedProfiles { get; set; }
    
```
I would leave an extra line between the dbsets and the next function